static/src/javascripts/projects/common/modules/commercial/msie-audit.js

```diff
@@ -0,0 +1,44 @@
+define([
+    'bonzo',
+    'common/utils/$',
+    'common/utils/config',
+    'common/modules/analytics/beacon',
+    'common/utils/detect',
+    'common/utils/_'
+], function (
+    bonzo,
+    $,
+    config,
+    beacon,
+    detect,
+    _
+) {
+
+    function init() {
+        if (!config.switches.msieAudit) {
+            return false;
+        }
+
+        // if I am MSIE, send a beacon
+        if (detect.getUserAgent.browser === 'MSIE') {
+            (_.once(function () {
+                beacon.beaconCounts('msie-browser');
+            }) ());
+
+            (_.delay(function () {
+                if (detect.adblockInUse) {
+                    beacon.beaconCounts('msie-adblock');
+                }
+            }, 5000) ());
+        }
+
+        // then set a timer to send ad detection beacon after 2 seconds
```

Comment:
I don't get it.

`_.delay` is the equivalent of `setTimeout`, it does not return a function, why are you using an IIFE?